src/components/lowering_replay.js

```diff
@@ -312,16 +323,16 @@ class LoweringReplay extends Component {
       return (
         <Card className="border-secondary p-1">
           <div className="d-flex align-items-center justify-content-between">
-              <span className="text-primary">00:00:00</span>
+              <span className="text-primary text-nowrap">{(this.state.utcScrubber ? this.state.lowering.start_ts:'00:00:00')}</span>
```

Comment:
Can this and the next change share logic with `sliderTooltipFormatter` so we don't need it in two places?

Reply:
Hm not sure about this one, maybe later we could pull out the renderControlsCard as a component and have that check the state for `useAbsoluteTimestamp`?